packages/bitcoin-node-wallet-provider/lib/BitcoinNodeWalletProvider.ts

```diff
@@ -73,7 +73,7 @@ export default class BitcoinNodeWalletProvider extends WalletProvider {
 
   async sendTransaction(options: SendOptions) {
     return options.fee
-      ? this.withTxFee(async () => this._sendTransaction(options), options.fee)
+      ? this.withTxFee(async () => this._sendTransaction(options), options.fee as number)
```

Comment:
Hmm weird, why do we need to cast `as number` everywhere? It looks like the places you are passing this, the type is already `number` and the destination type is `number`